### ISSUE TITLE
using semicolons as delimiters in less mixins

### ISSUE THREAD

**Issue by kalley** [csscomb/csscomb.js#393]
I'm getting
```
FATAL ERROR: CALL_AND_RETRY_LAST Allocation failed - process out of memory
Abort trap: 6
```

when using semicolons as delimiters in less files, as below:

```less
.button(@background; @border) {
     ...
}
```

**Comment by mradionov** [comment]
Having the same issue, switched to commas for now.

```
.button(@background, @border) {
     ...
}
```

**Comment by mradionov** [comment]
Current version of *gonzales-pe* for *csscomb* is **3.0.0-28**. In this version *gonzales-pe* fails to parse mixins. I am not sure about this **3.0.0-28** version whether it is stable or not, there is no even a tag for it on github, but it seems that it was published before **3.0.0** and **3.0.0** parses mixins fine.

**Comment by tonyganch** [comment]
Hey guys, the issue is fixed in the latest version of csscomb.
